src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -228,6 +228,8 @@ namespace AirflowNetworkBalanceManager {
 
     void ValidateDistributionSystem();
 
+    void ValidateFanFlowRate(); // Catch a fan flow rate from EPlus input file and add a flag for VAV terminal damper
```

Comment:
I don't even really think it needs to be exposed in the header; you could leave it contained in the implementation file, although that's rarely done in EnergyPlus...for now.